pkg/p2p/kadcast/bucket.go

```diff
@@ -0,0 +1,99 @@
+package kadcast
+
+// MAX_BUCKET_PEERS represents the maximum
+//number of peers that a `Bucket` can hold.
+const MAX_BUCKET_PEERS uint8 = 25
```

Comment:
I agree, it should be parametrized, that's why I implemented the main.go to simulate there the setup of `K` and `alpha`.

Anyway, until the end of the development I think it will be easier to keep it with the `var` definition and then change it.